autoware_launch/rviz/autoware.rviz

```diff
@@ -1011,6 +1011,98 @@ Visualization Manager:
                         Constant Color: false
                         Scale: 0.30000001192092896
                         Value: true
+                    - Class: rviz_plugins/Path
+                      Color Border Vel Max: 3
+                      Enabled: false
+                      Name: PathReference_Avoidnace
+                      Topic:
+                        Depth: 5
+                        Durability Policy: Volatile
+                        History Policy: Keep Last
+                        Reliability Policy: Reliable
+                        Value: /planning/path_reference/avoidance
+                      Value: true
+                      View Path:
+                        Alpha: 0.3
+                        Color: 210; 110; 210
+                        Constant Color: true
+                        Value: true
+                        Width: 2
+                      View Velocity:
+                        Alpha: 0.30000001192092896
+                        Color: 0; 0; 0
+                        Constant Color: false
+                        Scale: 0.30000001192092896
+                        Value: false
+                    - Class: rviz_plugins/Path
+                      Color Border Vel Max: 3
+                      Enabled: false
+                      Name: PathReference_LaneChange
+                      Topic:
+                        Depth: 5
+                        Durability Policy: Volatile
+                        History Policy: Keep Last
+                        Reliability Policy: Reliable
+                        Value: /planning/path_reference/lane_change
+                      Value: true
+                      View Path:
+                        Alpha: 0.3
+                        Color: 210; 110; 210
```

Comment:
@satoshi-ota 
As the path reference remains constant in color, it may be nicer to differentiate the color between each module. What are your thoughts on this matter?

Reply:
I agree with you. :+1:  Could you fix path color? :pray:  Please feel free to push this branch if necessary.

Reply:
Done in https://github.com/autowarefoundation/autoware_launch/pull/245/commits/0dea777afd3fa950e90aab4e0bb38b48efccb2bd!

Reply:
Thanks :+1: